src/utilities/units.ts

```diff
@@ -96,7 +96,7 @@ export function getVariablePrecision(value: number): number {
     // precision.
 
     let decimalPlacement = 0.1;
-    let precision = 1;
+    let precision = 3;
```

Comment:
@jrhosk I would prefer we add a comment here, explaining that the precision here is 3 instead of 1 because we add extra 2 digits for better UX.
Could you also update this branch with the latest dev, and add an item to `Added` in CHANGELOG.md?